packages/components/src/ConfirmationModal/ConfirmationModal.mdx

```diff
@@ -33,7 +33,7 @@ import {
         <Button label="Open" onClick={() => setOpen(true)} />
         <ConfirmationModal
           title="Should we?"
-          message="Do something…"
+          message={"### Do something..."}
```

Comment:
good example here. As a note, we try to keep the examples to a more realistic approach. In the case of `ConfirmationModal`, we would not likely have a heading in the message. Something like `message={`Let's **do something**`}` may be more realistic.

Reply:
Good change, I will update it.